zebrad/tests/acceptance.rs

```diff
@@ -202,6 +219,10 @@ fn app_no_args() -> Result<()> {
 
     output.stdout_contains(r"USAGE:")?;
 
+    // Make sure we have no info message in output
+    let notfound = output.stdout_contains(r"INFO");
+    assert!(notfound.is_err());
```

Comment:
I don't think there's a good way to check for any log message at any log level. And I don't think we want to match the exact usage message.

So let's delete this check.